charts/victoria-metrics-gateway/templates/ingress.yaml

```diff
@@ -0,0 +1,51 @@
+{{- if .Values.ingress.enabled }}
+{{- $ingressApiIsStable := eq (include "vmauth.ingress.isStable" .) "true" -}}
+{{- $ingressSupportsIngressClassName := eq (include "vmauth.ingress.supportsIngressClassName" .) "true" -}}
+{{- $ingressSupportsPathType := eq (include "vmauth.ingress.supportsPathType" .) "true" -}}
```

Comment:
this templates are missing from `_helpers.tpl`

also as @f41gh7 mentioned we should also use unique name like `vmgtw.ingres..`